src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -519,6 +468,10 @@ protected DataObject getJSON()
                 obj.putNull("nonce");
             else
                 obj.put("nonce", nonce);
+            if (components == null)
```

Comment:
It wasn't setup in this PR, but this method should really be called `toData`

Reply:
Out of scope